### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by phoob**
Ref #1

### CODE REVIEW

**Comment by ostark** [comment]
Thanks for providing the patch. @phoob  
this `getenv('PATH_PHP_BINARY') ?: '/usr/bin/php';` should fix it as well - `??`vs `?:`

File: src/Plugin.php [hunk 1]
```diff
@@ -79,13 +96,19 @@ protected function startBackgroundProcess()
...
-        $cmd    = $this->getBackgroundCommand($cmd);
-        $binary = getenv('PATH_PHP_BINARY') ?? '/usr/bin/php';
-
-        return sprintf($cmd, $binary);
+        $executableFinder = new PhpExecutableFinder();
+        if (false === $php = $executableFinder->find(false)) {
+            return null;
+        } else {
+            $cmd = array_merge(
+                array('nice', $php),
```
Comment (ostark) [inline src/Plugin.php]: can you explain this?
I don't get it.

File: src/Plugin.php [hunk 2]
```diff
@@ -79,13 +96,19 @@ protected function startBackgroundProcess()
...
+                $executableFinder->findArguments(),
```
Comment (ostark) [inline src/Plugin.php]: not really needed?
https://github.com/symfony/process/blob/master/PhpExecutableFinder.php#L73

Comment (phoob) [reply src/Plugin.php]: Well, it’s the way it’s done in https://github.com/symfony/process/blob/master/PhpProcess.php .. seems safer than relying on the env var, don’t you think?

Comment (ostark) [reply src/Plugin.php]: using PhpExecutableFinder is totally fine, but why $executableFinder->findArguments()?

Comment (phoob) [reply src/Plugin.php]: Well, it seems it takes into account arguments passed to the current php process, and reuses them in the new process it spawns.. But I don't know if it's necessary.

Comment (phoob) [reply src/Plugin.php]: and the 'nice' thing is needed for me – I have a setup that can generate something like 40 jobs at once, and therefore it's good to not make them steal all the juice.

Comment (phoob) [reply src/Plugin.php]: but maybe that doesn't work in windows..?

Comment (ostark) [reply src/Plugin.php]: `nice` does not work on :-)
Anyways - thanks for your help!

**Comment by ostark** [comment]
Thanks Peter!

**Pull request merged**